tools/src/main/java/com/nedap/archie/rules/evaluation/evaluators/BinaryTemporalAmountOperandEvaluator.java

```diff
@@ -0,0 +1,116 @@
+        }
+    }
+
+    /**
+     * Evaluate values that are an instance of TemporalAmount (Currently supported: Period and Duration)
+     *
+     * @param operator the operator of the expression
+     * @param left left value
+     * @param right right value
+     *
+     * @return A {@link Boolean} result of the comparison
+     */
+    private Boolean evaluateBooleanRelOp(OperatorKind operator, TemporalAmount left, TemporalAmount right) {
+        if (!left.getClass().equals(right.getClass())) {
+            throw new IllegalArgumentException("non matching classes not supported: " + left.getClass().getSimpleName() + " and " + right.getClass().getSimpleName() + ".");
+        }
+
+        switch (operator) {
+            case ge:
+                if (isEqual(left, right)) return true;
+            case gt:
+                return isGreater(left, right);
+            case le:
+                if (isEqual(left, right)) return true;
+            case lt:
+                return isLess(left, right);
+            case eq:
+                return isEqual(left, right);
+            case ne:
+                return !isEqual(left, right);
+            default:
+                throw new IllegalArgumentException("Not a boolean operator with boolean operands: " + operator);
+        }
+    }
+
+    private Boolean isLess(TemporalAmount left, TemporalAmount right) {
+        if (left instanceof Period && right instanceof Period) {
+            return periodToDaysEstimate((Period) left) - periodToDaysEstimate((Period) right) < 0;
+        } else if (left instanceof Duration && right instanceof Duration) {
+            return ((Duration) left).minus((Duration) right).isNegative();
+        } else {
+            throw new IllegalArgumentException("TemporalAmount class not supported: " + left.getClass().getSimpleName());
+        }
+    }
+
+    private Boolean isGreater(TemporalAmount left, TemporalAmount right) {
+        if (left instanceof Period && right instanceof Period) {
+            return periodToDaysEstimate((Period) left) - periodToDaysEstimate((Period) right) > 0;
+        } else if (left instanceof Duration && right instanceof Duration) {
+            return ((Duration) right).minus((Duration) left).isNegative();
+        } else {
+            throw new IllegalArgumentException("TemporalAmount class not supported: " + left.getClass().getSimpleName());
+        }
+    }
+
+    private Boolean isEqual(TemporalAmount left, TemporalAmount right) {
+        if (left instanceof Period && right instanceof Period) {
+            return periodToDaysEstimate((Period) left) - periodToDaysEstimate((Period) right) == 0;
+        } else if (left instanceof Duration && right instanceof Duration) {
+            return ((Duration) right).minus((Duration) left).isZero();
+        } else {
+            throw new IllegalArgumentException("TemporalAmount class not supported: " + left.getClass().getSimpleName());
```

Comment:
Archie supports PeriodDuration, which is the ISO-standard duration class, with both days and hours. That really should be supported here as well, even though that is slightly more tricky to do.